web/themes/custom/mytheme/src/scss/social_media.scss

```diff
@@ -0,0 +1,49 @@
+@import "mixin";
+@import "variables";
```

Comment:
they are already included in the Global.scss, isn't?

Reply:
+